packages/preload/src/index.ts

```diff
@@ -245,9 +246,12 @@ function initExposure(): void {
   contextBridge.exposeInMainWorld('restartPod', async (engine: string, podId: string): Promise<void> => {
     return ipcInvoke('container-provider-registry:restartPod', engine, podId);
   });
-  contextBridge.exposeInMainWorld('generatePodmanKube', async (engine: string, names: string[]): Promise<string> => {
-    return ipcInvoke('container-provider-registry:generatePodmanKube', engine, names);
-  });
+  contextBridge.exposeInMainWorld(
+    'generatePodmanKube',
+    async (engine: string, names: string[], kubeGeneratorId?: string): Promise<string> => {
+      return ipcInvoke('container-provider-registry:generatePodmanKube', engine, names, kubeGeneratorId);
+    },
```

Comment:
should we add for now a new method that return GenerateKubeResult instead of string (and mark this method as deprecated)

then update rendering part to the new API and remove this method after ?

Reply:
I've a question: should we not keep the method as before ? (so no kubeGeneratorId for now) until the renderer part switch to the new method